backend/schema/room.py

```diff
@@ -1,15 +1,35 @@
 from datetime import datetime
+from enum import Enum
 from typing import Dict, Optional
 
-from pydantic import BaseModel
+from fastapi import HTTPException
+from pydantic import BaseModel, root_validator, validator
+
+
+class Role(str, Enum):
+    """
```

Comment:
😁  The docstring needs a little adjustment
```suggestion
    """Enum for the roles of a user in a room."""
```

Reply:
Fixed